src/foam/nanos/approval/ApprovalRequest.js

```diff
@@ -166,7 +166,7 @@ foam.CLASS({
       columnPermissionRequired: true
     },
     {
-      class: 'Long',
+      class: 'String',
```

Comment:
Does this load existing approval requests?

Reply:
No, it doesn't, we would need to run the migration script.

Reply:
Actually, the replay works fine and the old data with long ids are loaded successfully.